src/cpuinfo_x86.c

```diff
@@ -125,6 +125,41 @@ static bool HasTmmOsXSave(uint32_t xcr0_eax) {
                                MASK_ZMM16_31 | MASK_XTILECFG | MASK_XTILEDATA);
 }
 
+static bool HasSecondFMA(uint32_t model) {
+    // Skylake server
+    if (model==0x55) {
+        char proc_name[49] = {0};
+        FillX86BrandString(proc_name);
+        // detect Xeon
+        if (proc_name[9]=='X') {
+            // detect Silver or Bronze
+            if (proc_name[17]=='S' || proc_name[17]=='B') {
```

Comment:
We don't use braces for single line statements (here and below)

Reply:
Okay.  I use curly braces compulsively as a result of https://blog.codecentric.de/en/2014/02/curly-braces/ but will violate that if it is required for this project.